docs/Rakefile

```diff
@@ -8,8 +8,8 @@ task :build do
 end
 
 desc "Run Markdownlint to validate the Markdown style."
-task lint: :build do
-  sh "mdl", "."
+task :lint do
+  sh "mdl $(git ls-files '*.md' | grep -v 'Manpage.md')"
```

Comment:
Is there any reason why we can't just hardcode the location of `Manpage.md`? It's already hardcoded [here](https://github.com/Homebrew/brew/blob/68b944a94fc1a807335aeed79576a7fe34de32e2/Library/Homebrew/dev-cmd/generate-man-completions.rb#L40) and hasn't been moved in the five years since [it was first added](https://github.com/Homebrew/brew/pull/2383).

Reply:
@apainintheneck this is ignoring the `Manpage.md`, presumably because it's generated.

Reply:
Of course 🤦